### PR TITLE
feat(http): add createAt & updatedAt variable

### LINKED ISSUE #13485: Variable: Add a createdAt to each variable
as a user
with admin
when i view the list of all variables
then i would like to view when each was created

### LINKED ISSUE #13486: Variable: Add a lastUpdatedAt to each variable
as a user
with admin
when i view the list of all variables
then i would like to see when the last update was

### PR DESCRIPTION
**Pull request by lisale0**
Closes #13485 
Closes #13486 

Describe your proposed changes here.

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [x] http/swagger.yml updated (if modified Go structs or API)
- [x] Documentation updated or issue created (provide link to issue/pr)
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: http/variable_service.go [hunk 1]
```diff
@@ -234,25 +233,24 @@ func newVariableResponse(m *platform.Variable, labels []*platform.Label) variabl
 	for _, l := range labels {
 		res.Labels = append(res.Labels, *l)
 	}
-
+	fmt.Print(m)
```
Comment (kelwang) [inline http/variable_service.go]: remove this

File: http/swagger.yml [hunk 2]
```diff
@@ -6883,6 +6883,9 @@ components:
             - $ref: "#/components/schemas/QueryVariableProperties"
             - $ref: "#/components/schemas/ConstantVariableProperties"
             - $ref: "#/components/schemas/MapVariableProperties"
+        createdAt:
+          type: string
+          format: date-time
```
Comment (kelwang) [inline http/swagger.yml]: pls also add updatedAt

File: http/variable_service.go [hunk 3]
```diff
@@ -234,25 +233,24 @@ func newVariableResponse(m *platform.Variable, labels []*platform.Label) variabl
...
-
 	req, err := decodePostVariableRequest(ctx, r)
 	if err != nil {
 		EncodeError(ctx, err, w)
 		return
 	}
+	va := req.variable
 
-	err = h.VariableService.CreateVariable(ctx, req.variable)
+	err = h.VariableService.CreateVariable(ctx, va)
```
Comment (kelwang) [inline http/variable_service.go]: va can be replaced back to req.variable

**Comment by jademcgough** [comment]
This also closes #13486

**Pull request merged**